setup.py

```diff
@@ -52,37 +49,11 @@ def run(self):
 
 
 setup(name = "plumbum",
-    version = version_string,  # @UndefinedVariable
-    description = "Plumbum: shell combinators library",
-    author = "Tomer Filiba",
-    author_email = "tomerfiliba@gmail.com",
-    license = "MIT",
-    url = "https://plumbum.readthedocs.io",
-    packages = ["plumbum", "plumbum.cli", "plumbum.commands", "plumbum.machines", "plumbum.path", "plumbum.fs", "plumbum.colorlib"],
-    package_data={
-        "plumbum.cli":["i18n/*/LC_MESSAGES/*.mo"]
-    },
-    platforms = ["POSIX", "Windows"],
-    provides = ["plumbum"],
-    keywords = "path, local, remote, ssh, shell, pipe, popen, process, execution, color, cli",
     cmdclass = {'test':PyTest,
                 'docs':PyDocs},
-    long_description = open(os.path.join(HERE, "README.rst"), "r").read(),
-    classifiers = [
-        "Development Status :: 5 - Production/Stable",
-        "License :: OSI Approved :: MIT License",
-        "Operating System :: Microsoft :: Windows",
-        "Operating System :: POSIX",
-        "Programming Language :: Python :: 2.6",
-        "Programming Language :: Python :: 2.7",
-        "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.4",
-        "Programming Language :: Python :: 3.5",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
-        "Topic :: Software Development :: Build Tools",
-        "Topic :: System :: Systems Administration",
-    ],
+    use_scm_version={
```

Comment:
I think I'd prefer the PEP 517 method of setuptools_scm; I'm pretty sure the produced wheel is still valid on older pip's, and that way on a build system you can ensure the correct versions of things (non-wheel installs would require a recent pip). Thoughts?